client/reader/following/main.tsx

```diff
@@ -71,6 +72,20 @@ function FollowingStream( { ...props } ) {
 					>
 						<ViewToggle />
 					</NavigationHeader>
+					<Banner
+						callToAction={ translate( 'Take the survey' ) }
+						description={ translate(
+							'Got a minute? Share feedback to help shape WordPress.com for creators and content consumption in 2026.'
+						) }
+						dismissPreferenceName="reader-creator-survey-2026-banner"
+						dismissTemporary
+						horizontal
+						href="https://automattic.survey.fm/creating-consuming-on-wordpress-com"
+						title={ translate( 'Help shape WordPress.com for creators' ) }
+						tracksImpressionName="calypso_reader_creator_survey_banner_view"
+						tracksClickName="calypso_reader_creator_survey_banner_click"
+						tracksDismissName="calypso_reader_creator_survey_banner_dismiss"
```

Comment:
I don't seem to be able to trigger the tracks events when clicking or dismissing the banner. We would need to add an event prop for it to work:


```suggestion
						event="reader_creator_survey_2026"
						tracksImpressionName="calypso_reader_creator_survey_banner_view"
						tracksClickName="calypso_reader_creator_survey_banner_click"
						tracksDismissName="calypso_reader_creator_survey_banner_dismiss"
```